### ISSUE TITLE
Active admin is creating 2 submit buttons

### ISSUE THREAD

**Issue by djalmaaraujo** [activeadmin/activeadmin#5735]
![image](https://user-images.githubusercontent.com/3402/55993308-230d7100-5c85-11e9-98ea-f92def5ed8f3.png)

### Expected behavior

It should have only one button rendered

### Actual behavior

For some reason, 2 buttons are being displayed

### How to reproduce

```ruby
form html: { multipart: true } do |f|
    f.inputs do
      f.input :user, label: "Usuário"
      f.input :status, as: :select, collection: Apuration.status.options
      f.input :due_date, label: "Prazo", as: :date_time_picker
      f.input :email_sent, label: "E-mail enviado?"
      f.input :brokeage_notes_files, as: :file, input_html: { multiple: true }
      f.input :balance_files, as: :file, input_html: { multiple: true }
      f.input :apuration_client_file, as: :file
      f.input :apuration_report_file, as: :file
      f.input :apuration_tax_file, as: :file
      f.input :created_at, label: "Mês Referência", as: :date_time_picker
      f.input :tax_total
      f.input :plan, label: "Plano Apurado"
      f.input :payment_total, label: "Valor à ser pago (Sobrescreve o plano)"
      f.actions
    end
  end
```

Active admin version:

```
gem 'activeadmin', '~> 1.4.3'
gem 'activeadmin_addons'
```

**Comment by deivid-rodriguez** [comment]
This is most likely https://github.com/activeadmin/arbre/issues/119. Are you using arbre 1.2.0, does it get fixed if you downgrade to 1.1.1?

**Comment by djalmaaraujo** [comment]
Thanks I will close this.
